galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.selenium.manager/bnd.bnd

```diff
@@ -9,11 +9,16 @@ Import-Package: dev.galasa,\
     dev.galasa.framework.spi,\
     dev.galasa.framework.spi.cps,\
     dev.galasa.framework.spi.language,\
+    dev.galasa.http.manager,\
+	dev.galasa.docker.manager,\
+	dev.galasa.kubernetes.manager,\
+	dev.galasa.artifact.manager,\
     dev.galasa.selenium,\
     javax.net,\
     javax.net.ssl,\
     org.apache.commons.logging,\
-    org.openqa.selenium
+    org.openqa.selenium,\
+    org.openqa.selenium.*
```

Comment:
Not happy with that.   you are exporting 3 packages above,  so you can simply import those 3 back in.    none of the openqa packages are exported from anything else in the OBR